docs/core_docs/docs/integrations/llms/google_palm.mdx

```diff
@@ -28,3 +29,100 @@ the model.
 import GooglePaLMExample from "@examples/models/llm/googlepalm.ts";
 
 <CodeBlock language="typescript">{GooglePaLMExample}</CodeBlock>
+
+# Google Vertex AI
```

Comment:
See above - why are we modifying legacy pages?

Reply:
They're not actually getting modified. The previous vertex page had docs on what is now legacy plam code, so I'm moving the content around so it's all on the proper pages now. (The headers were wrong though)